From 6b73a2f4622a23a51dc4df578f5c345397bc5fda Mon Sep 17 00:00:00 2001
From: Jan Palus <jpalus@fastmail.com>
Date: Wed, 10 Apr 2024 19:15:15 +0200
Subject: [PATCH] Fix python path when invoking py-compile

48eebb2 replaced custom PYTHON3 variable with PYTHON by using standard
AM_PATH_PYTHON macro. Makefile however still referred to old one.
There's no need to set PYTHON explicitly anymore so drop it.

Fixes #53
---
 bindings/python3/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/bindings/python3/Makefile.am b/bindings/python3/Makefile.am
index 70a1dd8..6072fc2 100644
--- a/bindings/python3/Makefile.am
+++ b/bindings/python3/Makefile.am
@@ -27,7 +27,6 @@ AM_CPPFLAGS = -I. -I$(top_builddir) $(PYTHON3_INCLUDES)
 LIBS = ${top_builddir}/src/libcap-ng.la
 SWIG_FLAGS = -python
 SWIG_INCLUDES = ${AM_CPPFLAGS}
-PYTHON = $(PYTHON3)
 pyexec_PYTHON = capng.py
 pyexec_LTLIBRARIES = _capng.la
 pyexec_SOLIBRARIES = _capng.so
